apiary.apib

```diff
@@ -0,0 +1,167 @@
+FORMAT: 1A
```

Comment:
Does this file need to be at the root, or can we place it in a `documentation` folder?